### DISCUSSION TITLE
playwright-reporter-github-actions/

### DISCUSSION THREAD

**Discussion by giscus** [estruyf/eliostruyf-comments#628] [Comments]
# playwright-reporter-github-actions/

Learn how to use the @estruyf/github-actions-reporter to discern Playwright test results in GitHub Actions quickly.

https://www.eliostruyf.com/playwright-reporter-github-actions/

**Comment by kassqa** [comment]
HI Elio. Thanks for the great work! I tried it though, however on github actions , I see this on the console Using GitHub Actions reporter. But I don't see the table as shown in the Doc.

This is how my config is 

 reporter: process.env.CI || true
    ? [
      ['html'],
      ['@estruyf/github-actions-reporter']
      ]
    : [["dot"], ["list"], ["html"]]
});

I have setup CI too. Any idea why this is the case?

**Reply by estruyf** [reply]
The only thing I can think of is that there is something going wrong during the tests or reporting. Is the HTML report correctly generated?
